packages/frontend/src/components/DashboardTiles/TileBase/TileBase.styles.tsx

```diff
@@ -0,0 +1,15 @@
+import { Colors } from '@blueprintjs/core';
+import styled from 'styled-components';
+
+export const ChartHeaderWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const FilterLabel = styled.span`
```

Comment:
Thanks for slowly moving the css into styled components. The main components look so clean without all the css. 🏅